File: belajar-loop/Belajar-loop1/loop.vb
'
'target :
'level 1
'****
'***
'**
'*
'
'Level2
'1
'22
'333
'4444
'
''Level3
'1
'12
'123
'1234

Imports System
Module Program
Sub Main()
	Dim a As Integer = 3
	For i As Integer = 0 To a
		For j As Integer = 0 To a-i
			Console.Write("*")
		Next j
		Console.WriteLine()
	Next i
	Console.WriteLine()
	For i As Integer = 0 To a
                For j As Integer = 0 To i
                        Console.Write(i+1)
                Next j
                Console.WriteLine()
        Next i
	Console.WriteLine()
        For i As Integer = 0 To a
                For j As Integer = 0 To i
                        Console.Write(j+1)                       
		Next j
                Console.WriteLine()
        Next i
End Sub
End Module

File: belajar-loop/Belajar-loop1/loop.js
/*
 * tantangan :
 * step harus positif
 *
 * Target :
 * Level1

 ****
 ***
 **
 *
 
 * level2
 
 1
 22
 333
 4444

 * Level3
 1
 12
 123
 1234
 */
/** @type {number} */
let a = 4;
for (let i=0;i<a;i++) {
	for (let j=0;j<a-i;j++) {
		process.stdout.write("*");
	}
	console.log();
}
console.log()
for (let i=0;i<a;i++) {
	for (let j=0;j<=i;j++) {
		process.stdout.write(`${i+1}`);
        }
        console.log();
}
console.log()
for (let i=0;i<a;i++) {
	for (let j=0;j<=i;j++){
		process.stdout.write(`${j+1}`);
	}
	console.log();
}
